l/lego/lego_ubi8.3.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/env bash
+# ----------------------------------------------------------------------------
+#
+# Package        : Lego
+# Version        : 3.5.0 - latest
+# Source repo    : https://github.com/go-acme/lego
+# Tested on      : Ubi 8.3 + Go latest version (v1.7.1)
+# Script License : Apache 2.0
+# Maintainer     : Eshant Gupta <eshant.gupta@ibm.com>; Priya Seth <sethp@us.ibm.com>
+#
+# Disclaimer: This script has been tested both in root and user mode on given
```

Comment:
To keep it simple, we can simply have this script run as root user. Being a _go_ package, it's not necessary to have a capability to run as non-root.